core/templates/pages/exploration-player-page/templates/take-break-modal.component.spec.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for TakeBreakModalComponent.
+ */
+
+import { TestBed } from
+  '@angular/core/testing';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+import { TakeBreakModalComponent } from
+  './take-break-modal.component';
+
+
+describe('TakeBreakModalComponent', function() {
+  let takebreakmodal: TakeBreakModalComponent;
+  let ngbActiveModal: NgbActiveModal;
+  beforeEach(() => {
+    takebreakmodal = TestBed.get(TakeBreakModalComponent);
```

Comment:
@atpug22  add the `Testbed.configureModule` and inject the dependencies.

Reply:
Done